src/contract-verification/sourcify/README.md

```diff
@@ -0,0 +1,13 @@
+# Sourcify
+
+Otterscan consumes verified smart contract source code and metadata from [Sourcify](https://sourcify.dev/).
+
+> Sourcify is open source, their data is public and they host a public instance of the verifier.
+
+Otterscan supports multiple ways to consume verified contract data.
+
+- [Integration modes](./integration-modes.md)
+
+If you are a smart contract developer who wants to verify a smart contract on Sourcify, please take a look at their docs. It is out of scope of this documentation to explain all different ways they can verify contracts.
```

Comment:
I strongly recommend removing the "it is out of scope" language altogether - that makes it sound like part of a technical report to deliver to a company or government, where scoping matters, rather than documentation. Let's just say "To learn how to verify a smart contract on Sourcify, take a look at the Sourcify docs: https://docs.sourcify.dev/docs/how-to-verify/"

Reply:
alright, pushed your suggestion